classes/classes/Creature.as

```diff
@@ -381,7 +381,7 @@ package classes
 		//MALE STUFF
 		//public var cocks:Array;
 		//TODO: Tuck away into Male genital class?
-		public var cocks:Array;
+		public var cocks:/*Cock*/Array;
```

Comment:
I have used `Vector` to store the Vagina instances, works without issues.
Since vectors can be accessed like arrays, the save / load code does not notice the difference.